### PR TITLE
batch_mode & interactive

### PR DESCRIPTION
**Pull request by AymenFJA**
related to #2779  and #2773

### CODE REVIEW

File: docs/source/configuration.ipynb [hunk 1]
```diff
@@ -126,7 +126,52 @@
...
+    "      `$ export RADICAL_PILOT_DBURL=mongodb://user:password@ip/db_name`\n",
+    "      \n",
+    "   - Run RP within these requested resources.\n",
+    "\n",
+    "      `$ python 00_getting_started.py`\n",
+    "   \n",
+    "- `batch_mode`: The native way of running applications on HPC is by creating a [batch_job](https://hpc-wiki.info/hpc/FAQ_Batch_Jobs) script describing the resource requirements for your application.\n",
+    "     For example, to run RP as a batch_job:\n",
+    "\n",
+    "   - Specify the `pd.access_schema = interactive`\n",
```
Comment (mtitov) [inline docs/source/configuration.ipynb:151]: `pd.access_schema = 'batch_mode'`

btw, maybe it should be just `batch`? (kinda we don't set `interactive_mode`)

Comment (andre-merzky) [reply docs/source/configuration.ipynb:151]: I agree with that.

Comment (AymenFJA) [reply docs/source/configuration.ipynb:151]: I agree as well.

**Review by mtitov — APPROVED** [review]
LGTM

**Pull request merged**